packages/playground/README.md

```diff
@@ -1,6 +1,6 @@
 # The playground
 
-The playground is the quickest way to hack on the internals of Theatre. It also hosts our end-to-end tests. It uses a simple vite config that builds all the related packages in one go, so you _don't_ have to run a bunch of build commands separately to start developing.
+The playground is the quickest way to hack on the internals of Theatre. It also hosts our end-to-end tests. It uses a build setup that builds all the related packages in one go, so you _don't_ have to run a bunch of build commands separately to start developing.
 
```

Comment:
This change makes the doc slightly less descriptive, but the build setup seems to have increased in complexity. Perhaps you can add an extra sentence or two to describe the build setup/stack/process?